recipes/slumber/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "slumber" %}
+{% set version = "0.7.1" %}
+{% set sha256 = "96899ce914f8d7862ce34f4e50af3a82e35588ab61b4912b85844479f4d177cb" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [py2k]
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  preserve_egg_dir: True
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - requests
```

Comment:
Even though it's pulled in through `requests`, could you add `python` as a `run` requirement?